certification/internal/operatorsdk/operatorsdk_test.go

```diff
@@ -0,0 +1,167 @@
+package operatorsdk
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/artifacts"
+)
+
+const (
+	testStdoutValue               = `{}`
+	testBundleValidateStdoutValue = `{"passed": true, "outputs": null}`
+)
+
+var _ = Describe("OperatorSdk", func() {
+	var tmpdir string
+	BeforeEach(func() {
+		var err error
+		tmpdir, err = os.MkdirTemp("", "preflight-test-artifacts-*")
+		Expect(err).ToNot(HaveOccurred())
+		DeferCleanup(os.RemoveAll, tmpdir)
+		artifacts.SetDir(tmpdir)
+	})
+	When("The Scorecard result is good", func() {
+		It("should succeed", func() {
+			operatorSdk := NewOperatorSdk("foo.image", fakeExecCommandSuccess)
+			_, err := operatorSdk.Scorecard(context.TODO(), "foo.image", OperatorSdkScorecardOptions{
+				ResultFile:     "success.txt",
+				OutputFormat:   "json",
+				Selector:       []string{"selector1", "selector2"},
+				Kubeconfig:     "/my/kubeconfig",
+				Namespace:      "awesome-namespace",
+				ServiceAccount: "this-service-account",
+				Verbose:        true,
+				WaitTime:       "120m",
+			})
+			Expect(err).ToNot(HaveOccurred())
+		})
+	})
```

Comment:
What are we testing here? This reads to me like we're evaluating the results of our scorecard execution to see if the asset passed. I know we're mocking here, but do we need to get an *operatorSDK to test that scorecard results have passed?

Reply:
It's basically the happy path, with all options set. Should probably figure out how to check that the actual options made it into the exec.